specs/config/config-resources.html

```diff
@@ -228,9 +228,9 @@ <h2>Supported Operations on Components</h2>
     <li>POST to the configuration LDPC for a component to create a new stream for that component, optionally specifying an initial baseline;
       if an initial baseline is not specified, a real or notional empty baseline is implied as the starting point for the stream</li>
   </ul>
-  <p>Servers SHOULD support the Prefer header for GET on LDPCs specifying PreferMinimalContainer:
+  <p>Servers SHOULD support the Prefer header for GET on LDPCs specifying PreferMinimalContainer:</p>
```

Comment:
Could you please enable prettier linter check in CircleCI config for CfgM spec, please?

Reply:
Will do, in the near future when I work on the spec itself. At the moment, I'm just merging the shapes to address that issue. The spec needs several things to being it up to date, including conformance clause work, using the right URIs and most recent ReSpec, etc.

Reply:
In other words, the current draft would seriously fail most checks!

Reply:
To ease the pain, I'd suggest to enable format on save via https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode. This requires you joining the dark side ofc and replacing Notepad++ with VS Code.

Let's merge now as is.

Finally, Prettier is only checking well-formedness and enforces formatting of the HTML code, not much more than that.